build/wrapper.cc

```diff
@@ -110,13 +110,18 @@ void initialize_worker(
                         application_error,
                         is_retryable_error);
 
-          RAY_CHECK(return_vec.size() == 1);
-
           // TODO: support multiple return values
           // https://github.com/beacon-biosignals/Ray.jl/issues/54
-          (*returns)[0].second = return_vec[0];
-          return Status::OK();
+          if (return_vec.size() == 1) {
+            (*returns)[0].second = return_vec[0];
+            return Status::OK();
+          } 
+          else {
+            auto msg = "Task returned too many values: " + std::to_string(return_vec.size());
+            return Status::Invalid(msg);
```

Comment:
unsure what type of `Status` we want to return here? I went with `Invalid` as it seemed the most generic

Reply:
`NotImplemented` seems appropriate